kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/MessageFormat.java

```diff
@@ -1,8 +1,18 @@
 package com.provectus.kafka.ui.serde.schemaregistry;
 
+import java.util.Optional;
+
 public enum MessageFormat {
   AVRO,
   JSON,
   PROTOBUF,
-  UNKNOWN
+  UNKNOWN;
+
+  public static Optional<MessageFormat> fromString(String typeString) {
```

Comment:
This one returns null if there's no such a value:
`org.apache.commons.lang3.EnumUtils#getEnum(MessageFormat.class, typeString)`